packages/devtools_app/lib/src/screens/debugger/codeview.dart

```diff
@@ -302,7 +304,7 @@ class _CodeViewState extends State<CodeView>
           enabled: widget.controller.scriptsHistory.hasScripts,
         ),
       ],
-      contentBuilder: (context, script) {
+      contentBuilder: (context, dynamic script) {
```

Comment:
solve problem without adding dynamic. you are stopping the static type analysis from catching issues.